bundle/manifests/activemq-artemis-operator.clusterserviceversion.yaml

```diff
@@ -2288,4 +2290,49 @@ spec:
   provider:
     name: ArtemisCloud
     url: https://artemiscloud.io
+  relatedImages:
```

Comment:
how does this become necessary with the new sdk? Or is it a change for a different reason?

Reply:
related images were moved to `activemq_artemis_manager_related_images.yaml`, see https://github.com/artemiscloud/activemq-artemis-operator/commit/0f5751601b55467424ca5bda05ea9b79213791c4